bases/br_me_rais/microdados_vinculos/table_config.yaml

```diff
@@ -0,0 +1,492 @@
+table_id: microdados_vinculos # AUTO GENERATED
+dataset_id: br_me_rais # AUTO GENERATED
+
+version: v1.0
+
+last_updated: 2020-12-24 # AUTO GENERATED
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.
+description: |  # REQUIRED
+    Microdados públicos dos vínculos de emprego na RAIS.
+
+# Qual organização/departamento/pessoa tratou os dados?
+# As vezes há um ponto intermediário entre os dados originais e subir na Base dos Dados.
+# Se essa pessoa é você, preencha abaixo com suas informações.
+treated_by:
+    name: <nome>  # REQUIRED
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_me_rais/code # REQUIRED
+    website: <onde encontrar os dados tratados>
+    email: <email>
+
+# Quem está completando esse arquivo config?
+published_by:
+    name: Ricardo Dahis
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_me_rais/code # REQUIRED
+    website: www.ricardodahis.com
+    email: rdahis@basedosdados.org
+
+# Se houve passos de tratamento, limpeza e manipulação de dados, descreva-os aqui.
+treatment_description: | 
+    Unificamos todos os anos consertando nomes de colunas, erros de valores, etc.
+
+# Com qual frequência a base é atualizada?
+# Opções: hora | dia | semana | mês | 1 ano | 2 anos | 5 anos | 10 anos | único | recorrente
+data_update_frequency: 1 ano
+
+# Nível da observação: quais colunas identificam uma linha unicamente?
+# Escolha todas as opções necessárias.
+# Regras: minúsculo, sem acento, singular.
+# Opções (exemplos): pais, estado, municipio, cidade, hora, dia, semana, mes, ano, etc.
+observation_level:
+    - vinculo
```

Comment:
Não existem essas colunas, resolver com os comentários anteriores

Reply:
Essa é uma situação nova pra gente. É uma base desidentificada que não inclui identificadores únicos de linha. Cada linha aqui representa um vínculo. Por isso botei o nível da observação assim.

Reply:
Se for esse o caso não faz sentido colocarmos uma coluna `id_vinculo` simplesmente como um índice de cada linha na tabela? Porque no modo original não existe nenhuma coluna que define unicamente cada linha, correto?

Reply:
Isso, a base original não tem nenhuma coluna identificando unicamente cada linha.

Vejo argumentos pró e contra botar a coluna de identificador.

- Pró: ter a coluna não machuca ninguém.
- Contra: (1) não é muito útil (porque não faz diferença identificar uma linha, porque vão ter observações com todas colunas repetidas), (2) não estão nos arquivo no storage agora (mas daria para gerar direto na query de view com uma combinação de ano + sigla_uf + número da linha), e (3) vai deixar mais pesada a base que já tem 260 GB.

Por mim não incluiria, mas sem preferência muito forte. Se achar fundamental eu acrescento. No mais, o resto tá todo resolvido e pronto pro merge.

Reply:
Algumas opções que penso aqui, sem mexer na tabela: 
- Deixar `vinculo` e indicar na descrição sobre a base ser desidentificada (coloquei como sugestão lá)
- Trocar para `vinculo (sem identificacao)`, por desencargo de consciência
- Deixar com todas as colunas - mas no caso podem existir ainda duas linhas iguais, e que devem ser tratadas como elementos distintos não?
- Deixar vazio, pois nenhuma coluna identifica uma linha unicamente.
- Deixarmos por padrão `index`, mesmo que não tenha na tabela

Levando em conta a descrição de Nível de observação aqui, até para definirmos para casos semelhantes no futuro.
```
# Nível da observação: quais colunas identificam uma linha unicamente?
# Escolha todas as opções necessárias.
# Regras: minúsculo, sem acento, singular.
# Opções (exemplos): pais, estado, municipio, cidade, hora, dia, semana, mes, ano, etc.
```

O que acha melhor?

Reply:
Boa. Acho que a discussão está em se vale a pena ou não separar, conceitualmente, o que o nível da observação e se há colunas que identificam unicamente cada linha. Tô achando que devemos sempre ter o primeiro, mas não necessariamente o segundo, de acordo com cada tabela.

Se formos com isso, realmente teremos que mudar o `table_config.yaml`. Sugiro quebrarmos em duas colunas: uma `nível da observação` e outra `colunas identificando linhas unicamente`. Nesse caso da RAIS vínculos, a primeira seria `vínculo` e a segunda poderia estar vazia.

Que tal?

Reply:
hnmnm é uma boa linha de pensamento... acho que fica suficiente adicionar um booleano `has_identifiers` para indicar se `observation_level ` consta na tabela como um identificador ou não.